relay.go

```diff
@@ -281,9 +281,22 @@ func (r *Relayer) Receive(f *Frame, fType frameType) (sent bool, failureReason s
 	case r.conn.sendCh <- f:
 	default:
 		// Buffer is full, so drop this frame and cancel the call.
-		r.logger.WithFields(
-			LogField{"id", id},
-		).Warn("Dropping call due to slow connection.")
+
+		// Since this is typically due to the send buffer being full, get send buffer
+		// usage + limit and add that to the log.
+		sendBuf, sendBufLimit, sendBufErr := r.conn.sendBufSize()
+		logFields := []LogField{
+			{"id", id},
+			{"destConnSendBufferCurrent", sendBuf},
+			{"destConnSendBufferLimit", sendBufLimit},
+			{"sendChQueued", len(r.conn.sendCh)},
```

Comment:
Perhaps also `sendChQueuedBytes`? That could hint if it's many frames in a short time window ([microburst](https://en.wikipedia.org/wiki/Micro-bursting_(networking))) vs. fragmentation of a large response.

I realize there's coordination overhead associated with this kind of accounting... Might be not worth it.

Reply:
Yeah this would require a larger accounting change, given that we get the numbre of bytes in the TCP send buffer, I think we should start with that.

If that's not enough, we can then add tracking for `sendChQueuedBytes` as a separate change, but I'd prefer to avoid that complexity for now.